src/main/java/uk/gov/companieshouse/account/validator/service/handler/request/status/RequestStatusHandler.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.companieshouse.account.validator.service.handler.request.status;
+
+import java.time.LocalDateTime;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import uk.gov.companieshouse.account.validator.model.felix.ixbrl.Results;
+import uk.gov.companieshouse.account.validator.model.validation.RequestStatus;
+import uk.gov.companieshouse.account.validator.repository.RequestStatusRepository;
+import uk.gov.companieshouse.api.model.felixvalidator.ValidationStatusApi;
+
+@Component
+public class RequestStatusHandler {
```

Comment:
Can we call this RequestStatusFactory. A Handler normally refers to a request handler